.travis.yml

```diff
@@ -221,15 +221,15 @@ matrix:
         - postgresql
       env: DB=pgsql POSTGRESQL_VERSION=9.4
     - php: hhvm
-      sudo: false
+      sudo: true
       dist: trusty
       addons:
         postgresql: "9.5"
       services:
         - postgresql
       env: DB=pgsql POSTGRESQL_VERSION=9.5
     - php: hhvm
-      sudo: false
+      sudo: true
```

Comment:
Yes, see the listed HHVM issues in the PR statement. It's due to the type annotations issue.